lib/livebook_web/live/output/frame_component.ex

```diff
@@ -0,0 +1,60 @@
+defmodule LivebookWeb.Output.FrameComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def mount(socket) do
+    {:ok, assign(socket, counter: 0)}
+  end
+
+  @impl true
+  def update(assigns, socket) do
+    {update_type, assigns} = Map.pop(assigns, :update_type, nil)
+    {outputs, assigns} = Map.pop!(assigns, :outputs)
+
+    socket = assign(socket, assigns)
+
+    socket =
+      case update_type do
+        nil ->
+          assign(socket, outputs: outputs)
+
+        :replace ->
+          socket |> assign(outputs: outputs) |> update(:counter, &(&1 + 1))
```

Comment:
You don't need the counter if you are not using phx-append I think? 🤔

Reply:
We add counter to child output `id`s, this way when we replace output in the frame we make sure the new one is rendered (rather than falling under `phx-update="ignore"` of the previous one).